pkg/crc/preflight/preflight_darwin.go

```diff
@@ -2,14 +2,12 @@ package preflight
 
 import (
 	"fmt"
-	"os"
-	"path/filepath"
 
 	"github.com/code-ready/crc/pkg/crc/constants"
 	"github.com/code-ready/crc/pkg/crc/network"
 	"github.com/code-ready/crc/pkg/crc/preset"
 	crcpreset "github.com/code-ready/crc/pkg/crc/preset"
-	crcos "github.com/code-ready/crc/pkg/os"
+	"github.com/code-ready/crc/pkg/os/launchd"
```

Comment:
this already exists or is missing from the PR?

Reply:
This already exists https://github.com/code-ready/crc/blob/main/pkg/os/launchd/launchd_darwin.go
(should have been removed a while ago, but was forgotten, and it's being reused in https://github.com/code-ready/crc/pull/3072 so we'll keep it)